src/content/blog/dart-macros.md

```diff
@@ -0,0 +1,67 @@
+---
+title: On Focus and Dart Macros
+author: eseidel
+description: Reacting to Google's decision to cancel Dart's "macros" feature
+date: 2025-01-29
+---
+
+Brace yourselves. We may be in for another round of “Flutter is dying” and “Dart
+is dying” hot takes across the blog-o-sphere.
+
+But again, maybe at this point I’m used to it. I co-founded the Flutter project
+at Google in 2014 and led the Flutter (and later Dart) teams at Google until
+2022 when leaving to found this company, Shorebird. Shorebird sells products to
+teams using Flutter and I remain deeply involved and invested in Flutter and
+Dart's success. Flutter has been “dying” or “about to be canceled by Google”
+since we started the project 10 years ago. So I guess if "accounting for (1/3rd
+of app submissions to
+AppStore)[https://medium.com/flutter/flutter-in-production-f9418261d8e1] or
+Play” is what "dying" looks like, I’m here for it. :slightly_smiling_face: 
```

Comment:
I have no idea why it inserted "slightly smiling face" instead of the actual emoji :roll_eyes:

Reply:
Will try it and see if astro handles it, if it doesn't I"ll fix it.

Reply:
Yeah doesn't.  Fixing.